### PR TITLE
Solve #33

### PR DESCRIPTION
**Pull request by Mahir-G**
Remove empty lines from divyam3897. (Solve #33)

### CODE REVIEW

**Review by divyam3897 — CHANGES_REQUESTED** [review]
All contact info is not on next lines.

**Review by divyam3897 — CHANGES_REQUESTED** [review]
Just change the commit message and write a good one and it's good to be merged.

File: submissions/mahirgupta/about.md [hunk 1]
```diff
@@ -3,8 +3,17 @@ My name is Mahir Gupta. I am an IT student at Panjab University, India.
 # Experience
 I am a newbie in programming.
 # Interests
+<<<<<<< HEAD
```
Comment (sagar-kalra) [inline submissions/mahirgupta/about.md:6]: remove these conflicts

File: submissions/divyam3897/about.md [hunk 2]
```diff
@@ -1,23 +1,15 @@
...
-
-Linkedin: [Divyam Madaan](https://www.linkedin.com/in/divyam-madaan-0bb627105/)
-
-Github: [divyam3897](https://github.com/divyam3897)
-
+Facebook profile: [Divyam Madaan](https://www.facebook.com/divyam.madaan.1)  
+Linkedin: [Divyam Madaan](https://www.linkedin.com/in/divyam-madaan-0bb627105/)  
+Github: [divyam3897](https://github.com/divyam3897)  
 # Emoji
 My favorite emoji is :smile:
```
Comment (sagar-kalra) [inline submissions/divyam3897/about.md:15]: remove this extra line.

**Review by sagar-kalra — CHANGES_REQUESTED** [review]

**Comment by divyam3897** [comment]
Fixed in https://github.com/pclubuiet/Hacktoberfest-2k17/commit/83595182d1750e6d43f65943d71a129c497b9f05

**Pull request closed without merging**